permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -80,7 +173,12 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         }
 
         // Message
-        if (mMessage != null) {
+        if (getArguments().containsKey(KEY_HAS_HTML) && getArguments().getBoolean(KEY_HAS_HTML)) {
```

Comment:
No need to check ```containsKey()```, as ```getBoolean()``` defaults to false if the key isn't present. [Docs](https://developer.android.com/reference/android/os/BaseBundle.html#getBoolean(java.lang.String))